spec/board_maker_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+require 'board_maker'
+
+describe BoardMaker do
+  let(:maker) { BoardMaker.new }
+
+  context 'squares' do
+    it 'makes a square for a size 1 board' do
+      board = Board.new
+      expect(board.squares.count).to eq(0)
+      maker.populate_squares(board)
+      expect(board.squares.count).to eq(1)
+    end
+  end
+
+  context 'regions' do
```

Comment:
Think of your higher level `describe` box as describing the behaviour of your app - for example, this could be "making regions", with the specific conditions underneath. Always aim to make your specs read nicely!